lang/cs/Org.Apache.REEF.Driver/IDriverRestarted.cs

```diff
@@ -16,13 +16,19 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+
+using System;
+using System.Collections.Generic;
+
 namespace Org.Apache.REEF.Driver
 {
     /// <summary>
     /// Event fired on Driver restarts instead of IDriverStarted.
     /// </summary>
-    public interface IDriverRestarted : IDriverStarted
```

Comment:
The `StartTime` property could be shared with `IDriverStarted`, right? Hence, this inheritance might still make sense.